src/components/Footer.vue

```diff
@@ -48,12 +48,24 @@
           <a href="https://akcasc.org"
             >Alaska Climate Adaptation Science Center</a
           >, the University of Alaska Anchorage
-          <a
-            href="https://www.uaa.alaska.edu/academics/college-of-health/departments/population-health-sciences/institute-for-circumpolar-health-studies"
+          <a href="https://www.uaa.alaska.edu/academics/college-of-health/departments/population-health-sciences/institute-for-circumpolar-health-studies"
             >Institute for Circumpolar Health Studies</a
-          >, and the University of Washington
+          >, the University of Washington
           <a href="https://cdrc.uw.edu"
             >Center for Disaster Resilient Communities</a
+          >, Louden Tribe, the Native Village of Gakona, Igiugig Village Council, 
```

Comment:
```suggestion
          >, <a href="https://www.loudentribe.org/">Louden Tribe</a>, the <a href="https://nvgakona.com/">Native Village of Gakona</a>, <a href="https://www.igiugig.com/tribal-government/8-igiugig-tribal-council">Igiugig Village Council</a>, 
```

Not sure if we are deliberately omitting links for these organizations, but it looks like they all have websites/webpages we can link to. I defer to @MikeDelue on whether we should or not.

Reply:
I'd suggest trimming the igugig url at .../tribal-government instead of linking in to the council page. Otherwise these all look perfectly appropriate.